### PR TITLE
Add ELB HTTPS support

### PR DESCRIPTION
**Pull request by efinlay24**
Automagically enable HTTPS listener support for ELBs in CoCo delivery clusters.
Test cluster provisioned, working as expected.

### CODE REVIEW

**Comment by martingartonft** [comment]
I love this change. Nice one @efinlay24

File: README.md [hunk 1]
```diff
@@ -51,35 +51,13 @@ docker run \
     -e "API_HOST=$API_HOST" \
     -e "CLUSTER_BASIC_HTTP_CREDENTIALS=$CLUSTER_BASIC_HTTP_CREDENTIALS" \
     -e "ENVIRONMENT_TYPE=$ENVIRONMENT_TYPE" \
-    coco/coco-provisioner:v1.0.4
+    coco/coco-provisioner:v1.0.6
 
 ## If the cluster is running, set up HTTPS support (see below)
```
Comment (tosan88) [inline README.md]: Can you please remove this line as well?

Comment (efinlay24) [inline README.md]: Yup, done. Removed from pub provisioner readme as well, I missed it there.

Comment (tosan88) [inline README.md]: Thanks! :)

**Pull request merged**